### PR TITLE
ECER-2040/2035: View character and work experience reference pages

### PR DESCRIPTION
**Pull request by pcatkins**
## Description

- `ViewWorkExperienceReference` component
- `ViewCharacterReference` component
- Typography heading change

## Related Jira Issue(s)

- [ECER-1091](https://eccbc.atlassian.net/browse/ECER-1091)
- [ECER-1092](https://eccbc.atlassian.net/browse/ECER-1092)

## Checklist
- [X] I have tested these changes locally.
- [ ] Changes to backend endpoints are covered by passing integration tests.
- [ ] I have added or updated the necessary documentation.

## Screenshots (if applicable)
Screenshot 2024-05-30 at 4 31 00 PM
Screenshot 2024-05-30 at 4 30 49 PM

## Additional Comments (optional)
Any additional context or information that might be helpful for the reviewer.

### CODE REVIEW

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ViewCharacterReference.vue [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  setup: async (props) => {
+    const applicationStore = useApplicationStore();
+    const router = useRouter();
+
+    // Check store for existing reference
+    const reference: Components.Schemas.CharacterReference | undefined = applicationStore.characterReferenceById(props.referenceId);
+
+    if (!reference) {
+      router.back();
+    }
```
Comment (SoLetsDev) [inline src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ViewCharacterReference.vue:54]: I wonder if this should refer to the manage-applications page? ... we have the applicationId in the route. 

I'm thinking about the scenario if someone copy and pastes the link directly to the URL without any history to router.back to.

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ViewWorkExperienceReference.vue [hunk 2]
```diff
@@ -31,18 +51,14 @@ export default defineComponent({
...
-    const workExperienceReference = applicationStore.workExperienceReferenceById(props.referenceId);
+    // Check store for existing reference
+    const reference: Components.Schemas.WorkExperienceReference | undefined = applicationStore.workExperienceReferenceById(props.referenceId);
 
-    console.log(workExperienceReference);
-
-    if (!workExperienceReference) {
-      //   router.back();
+    if (!reference) {
+      router.back();
```
Comment (SoLetsDev) [inline src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ViewWorkExperienceReference.vue:58]: same comment as the character reference.

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ViewCharacterReference.vue (hunk 1 above)
Comment (pcatkins) [reply src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ViewCharacterReference.vue:54]: Wouldn't that be better to bounce back in that case? As it's not a healthy URL to hit when we don't have a valid reference link, we can't assume they have a valid application for `/manage-applications`

Comment (SoLetsDev) [reply src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ViewCharacterReference.vue:56]: hmm good point! I retract my statement.

**Review by SoLetsDev — APPROVED** [review]

**Pull request merged**